Tasks/ExtractFiles/extractFiles.ts

```diff
@@ -0,0 +1,307 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
```

Comment:
All strings need to be converted to lowercase before comparison.  Or, all uses of `endsWith()` need to be converted to a case-insensitive comparison, such as:
`file.toLowerCase().endsWith(".zip")`